### PR TITLE
top streams + load more + footer fixes

### LINKED ISSUE #4: Create component for top streams
Data for top streams is already fetched - abstract it to separate component and display it using bootstrap. Style it according to prototype.

### PR DESCRIPTION
**Pull request by Freshnek**
#4 #25

### CODE REVIEW

**Comment by Freshnek** [comment]
I was working on old version of repo, so I didn't have Home component, so this new top streams component need to be added there (Home.vue).

File: src/components/TopStreams.vue [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        );
+        this.streams = response.data.streams.map(stream => {
+          if (stream.channel.status.length > 50) {
+            let slicedStatus = stream.channel.status.slice(0, 50) + "...";
+            return { ...stream, slicedStatus };
+          }
+          else {
+              let slicedStatus = stream.channel.status
+            return { ...stream, slicedStatus };
+          }
```
Comment (bartek9105) [inline src/components/TopStreams.vue:91]: If I understand correctly, this just cuts text if it's too long?

Comment (Freshnek) [reply src/components/TopStreams.vue:91]: Ah that godlike css. Thanks I'll fix that

File: src/components/TopStreams.vue [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+          
+        });
+      } catch (error) {
+        console.log(error);
+      }
+    },
+    loadMore() {
+      this.streamsShown *= 2;
+    }
```
Comment (bartek9105) [inline src/components/TopStreams.vue:100]: Seems like you never use this?

File: src/components/TopStreams.vue [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    // setThumbnailSize() {
+
+    // }
+  },
```
Comment (bartek9105) [inline src/components/TopStreams.vue:104]: I would delete this

Comment (Freshnek) [reply src/components/TopStreams.vue:104]: true

**Comment by Freshnek** [comment]
Changes done. Please review

**Pull request merged**